src/icons/general/IconSearchJuicy.js.flow

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray40 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const IconSearchJuicy = ({ className = 'icon-search', color = bdlGray40, height = 14, title, width = 14 }: Icon) => (
```

Comment:
I think the default `className` value should be an empty string since a class is already specified on line 11. That seems to match how the other icon files do it.

Reply:
Hey @tjuanitas ! Thanks for the review. I just copied and pasted from `IconSearch`. Should we change, both, one, or neither?

Reply:
Probably both.

Reply:
Yeah probably both since it's seems more like an oversight rather than intentional.